src/lib/Usage/index.tsx

```diff
@@ -0,0 +1,7 @@
+import React from "react";
```

Comment:
There is no need of component, It should follow how we have introduction .. as in mdx format

Reply:
done, PR - https://github.com/ishaan-puniani/wallet-statistics/pull/832